controllers/controller.js

```diff
@@ -58,8 +55,23 @@ const postCommentsByArticleId = (req, res, next) => {
   .catch(next)  
 }
 
+const updateArticle = (req, res, next) => {
+
+  
+  const votedArticle = req.body.inc_votes; 
```

Comment:
If `req.body.inc_votes` doesn't exist for some reason (like a bad request has been made), we would need to reject the promise and send an error 
Also in terms of naming this variable, you're passing a number of votes, whereas I would assume the `votedArticle` would be an article object, you could rename this to something a bit clearer